package.json

```diff
@@ -2,7 +2,11 @@
   "name": "fixed-data-table-2",
   "version": "2.0.0-beta.7",
   "description": "A React table component designed to allow presenting thousands of rows of data.",
-  "main": "main.js",
+  "main": "dist/cjs/fixed-data-table-2.js",
+  "module": "dist/es/fixed-data-table-2.js",
+  "files": [
+    "dist"
```

Comment:
Whitelisted set of files to be published to npm (see [files](https://docs.npmjs.com/cli/v8/configuring-npm/package-json#files))